backend/src/server/routes/v1/index.ts

```diff
@@ -134,7 +135,13 @@ export const registerV1Routes = async (server: FastifyZodProvider) => {
   await server.register(registerIntegrationRouter, { prefix: "/integration" });
   await server.register(registerIntegrationAuthRouter, { prefix: "/integration-auth" });
   await server.register(registerWebhookRouter, { prefix: "/webhooks" });
-  await server.register(registerIdentityRouter, { prefix: "/identities" });
+  await server.register(
+    async (identitiesRouter) => {
+      await identitiesRouter.register(registerIdentityRouter);
+      await identitiesRouter.register(registerIdentityTemplateRouter, { prefix: "/templates" });
+    },
+    { prefix: "/identities" }
+  );
```

Comment:
I think identity templates should be its own resource and not nested in identities? like `/identity-templates`